package.json

```diff
@@ -4,6 +4,9 @@
   "version": "0.0.19",
   "author": "yurkimus <yurkimus@gmail.com>",
   "license": "ISC",
+  "scripts": {
+    "prepublishOnly": "node ./index.js"
```

Comment:
I think we should swap this script by generate and combine it with npx dprint fmt, so prepublishOnly brings combination of two scripts